Firestore/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Unreleased
 
+# v0.10.2
+- [changed] When you delete a FirebaseApp, the associated Firestore instances
+  are now also deleted (#683).
+- [fixed] Fixed race conditions in streams that could be exposed by rapidly
+  toggling the network from enabled to disabled and back (#772) or encountering
+  multiple successive failures from the server (#835).
+- [fixed] Addressed warnings shown by the latest versions of Xcode and CocoaPods.
```

Comment:
Went with the following for RTDB and Storage:

- [changed] Addresses CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF warnings that surface in newer versions of Xcode and CocoaPods.

Reply:
I only included this at all because you did that. If not for that I wouldn't have included this line item at all. In our case we addressed more than that (-Wcomma, -Wrange-loop-analysis) and it seems so far past overkill to enumerate them all.